bricks/constants_file/__brick__/constants.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter/material.dart';
+import 'dart:math';
+
+
+  /// This Consts file was generated using Mason. You can customize it to fit your needs.
+  /// Feel free to add any constants that you think will be useful for your project.
+class Consts {
+
+  /// The name of the app.
+  static const appName = '{{{app_name}}}';
+
+  /// Grid system's base value. The project uses an {{{grid_system}}} point grid system.
+  ///
+  /// This means that the minimum size must be {{{grid_system}}} and any other sizes must
+  /// be multiples of this value.
+  static const _gridSystem = {{grid_system}};
+
+  /// Default minimum padding size. The project uses an {{{grid_system}}} point
+  /// grid system.
+  ///
+  /// This means that the minimum padding size is {{{grid_system}}}, and any other sizes must
+  /// be multiples of this value.
+  static const padding = _gridSystem;
+
+  /// Default margin size for widgets and screens. The margin follows the
+  /// same grid system as the padding.
+  static const margin = padding * 2;
+
+  /// Default padding size to use between different sections.
+  ///
+  /// See [Section] widget for more information.
+  static const sectionPadding = padding * 4;
+
+  /// Default padding size to use between different fields.
+  ///
+  /// Usually used to separate [CustomField] widgets, but can be used
+  /// to separate any other widgets inside a list or column.
+  static const fieldPadding = padding * 2;
+
+  /// Default border radius value.
+  static const defaultRadius = {{default_radius}};
+
+  static const bottomPadding = _gridSystem * 5;
+
+  // <---------------- Colors ------------------>
+
+  static const Color fontWhite = Color(0xFF{{font_white}});
+  static const Color fontBlack = Color(0xFF{{font_black}});
+
+  static const Color _primaryOneSeedValue = Color(0xFF{{primary_color_seed_value}});
+  static final MaterialColor primaryOne = generateMaterialColor(_primaryOneSeedValue);
+  {{#project_includes_secondary_color}}
+  
+  static const Color _secondaryOneSeedValue = Color(0xFF{{secondary_color_seed_value}});
```

Comment:
En los bricks puedes usar condicionales para decidir si agregar algo al template o no. Por ejemplo, hasSecondaryColor (si es true, muestras las lineas 54:57. Si no, simplemente no agregas el color secundario.

Reply:
¿No está bien usarlo con la sintaxis de mostacho? Porque igual lo esta ocultando si el campo project_includes_secondary_color es falso. 👀

Reply:
You're right! No lo había visto 👀👀👀